lib/iteraptor.ex

```diff
@@ -435,15 +438,18 @@ defmodule Iteraptor do
       input,
       fn
         {k, [{_, _} | _] = kw} when is_list(k) ->
-          {k |> List.last() |> to_string(), jsonify(kw, opts)}
+          {k |> List.last() |> do_stringify(opts), jsonify(kw, opts)}
```

Comment:
I would avoid repetitive `Keyword.get/3` calls and would do instead:

```elixir
  stringify_keys = Keyword.get(opts, :keys, true)
  Iteraptor.map(
    ...
    {k |> List.last() |> do_stringify(stringify_keys), jsonify(kw, opts)}
```

and see below.